### PR TITLE
Ws 024 style homepage

### PR DESCRIPTION
**Pull request by DeepashriDali**
Hi Andy, Tim,
Could you please review the styling of the home page.
Deepa
image

### CODE REVIEW

File: frontend/src/components/homepage/recentSighting/RecentSighting.scss [hunk 1]
```diff
@@ -1,12 +1,89 @@
...
+.recent-sighting__right {
+  width: 30vw;
+  height: 40vh;
   display: flex;
   flex-direction: column;
   align-items: center;
+  justify-content: center;
+  text-align: center;
+  background-color: $color-background;
+  // background-color: rgba(130, 135, 138, 0.781);
```
Comment (AndrewYHuang) [inline frontend/src/components/homepage/recentSighting/RecentSighting.scss:57]: Residual comment?

File: frontend/src/components/homepage/popularLocations/PopularLocations.tsx [hunk 2]
```diff
@@ -10,18 +13,44 @@ export function PopularLocations() {
...
+              Plan a whale spotting trip!
+            </h2>
+            <Link to="/plantrip">
+              <Button
+                variant="light"
+                className="popular-locations__post-location-button"
+              >
+                <img
+                  src="https://freepngimg.com/download/icon/1000188-spouting-whale-emoji-free-icon-hq.png"
+                  width="30px"
```
Comment (AndrewYHuang) [inline frontend/src/components/homepage/popularLocations/PopularLocations.tsx:48]: Strictly, this should be an integer without a unit

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-width

File: frontend/src/components/homepage/recentSighting/RecentSighting.tsx [hunk 3]
```diff
@@ -27,20 +30,45 @@ export function RecentSighting() {
...
+              Report your whale sighting!
+            </h2>
+            <Link to="/sightings/create">
+              <Button
+                variant="light"
+                className="recent-sighting__post-sighting-button"
+              >
+                <img
+                  src="https://freepngimg.com/download/icon/1000188-spouting-whale-emoji-free-icon-hq.png"
+                  width="30px"
```
Comment (AndrewYHuang) [inline frontend/src/components/homepage/recentSighting/RecentSighting.tsx:66]: ditto here with the units

**Review by AndrewYHuang — CHANGES_REQUESTED** [review]
Looks good - there's a few nitpicks. Happy for you to merge after fixing

File: frontend/src/components/homepage/recentSighting/RecentSighting.scss (hunk 1 above)
Comment (DeepashriDali) [reply frontend/src/components/homepage/recentSighting/RecentSighting.scss:57]: Removed comment.

File: frontend/src/components/homepage/popularLocations/PopularLocations.tsx (hunk 2 above)
Comment (DeepashriDali) [reply frontend/src/components/homepage/popularLocations/PopularLocations.tsx:48]: Removed unit

File: frontend/src/components/homepage/recentSighting/RecentSighting.tsx (hunk 3 above)
Comment (DeepashriDali) [reply frontend/src/components/homepage/recentSighting/RecentSighting.tsx:66]: Removed unit

**Review by AndrewYHuang — APPROVED** [review]
Looks good!

**Pull request merged**